pages/planetusermanual.md

```diff
@@ -62,63 +62,63 @@ An example of the Manager Dashboard is shown below. The functionalities are defi
 
-14.**My Meetups** displays the physical location of the Course (if applicable).
+14. **My Meetups** displays the physical location of the Course (if applicable).
 
-15.**My Tutors** is under construction.
+15. **My Tutors** is under construction.
 
-16.**Badges** is under construction.
+16. **Badges** is under construction.
 
-17.**Calendar** can be used to organize and display Meetups and Events for the Learner.
+17. **Calendar** can be used to organize and display Meetups and Events for the Learner.
 
-18.**Email** is a basic email system which allows Learners to receive and set notifications and send and receive messages.
+18. **Email** is a basic email system which allows Learners to receive and set notifications and send and receive messages.
 
-19.**Surveys** allows Nation Managers to push a survey PLANET users based on characteristics like age, gender, member type, Community, etc.
+19. **Surveys** allows Nation Managers to push a survey PLANET users based on characteristics like age, gender, member type, Community, etc.
 
-20.**Update Welcome Video** allows Managers to change the welcome video for their Nation.
+20. **Update Welcome Video** allows Managers to change the welcome video for their Nation.
 
 # Library and Resources Overview
 
-## _What is the Library?_
+## What is the Library?
 
 The library where learning Resources are stored and categorized within the PLANET system.  Resources are listed alphabetized which allows Learners to select the first letter of their Resource or the page number they wish to view.  Additionally, library Resources can be organized into collections which allow Learners to categorize and group Resources by subject or other attributes.  Both Resources and collections are explained in further depth below.
 
-## _What is a Resource?_
+## What is a Resource?
 
 A PLANET Resource is material that has been uploaded into the digital library.  There are many types of Resources, not just books.  PLANET can accommodate PDFs (books), MP3s (audio), MP4s (videos), HTML5 applications  (interactive content), and just about any other format.  Resources can be viewed in the library and may vary from Nations to Communities.  Resources may also stored in a variety of languages, including (but not limited to) English, Kiswahili, French, Spanish, Arabic, and Somali.
 
-## _What is a Collection?_
+## What is a Collection?
 
 A collection is a group of Resources based on topic/subject and intended use.  PLANET currently follows a the Library of Congress categorization system.  Collections exist as the main category.  Sub-collections allow Nation Managers to group Resources more specifically within a collection.  Collections are alphabetized by topic.
 
 # How do I navigate and use the Library?
 
-## _How do I upload a Resource?_
+## How do I upload a Resource?
 
 1.  Log in as a **_Learner_**, **_Leader_**, or **_Manager_**.
```

Comment:
@aurinsomnia I think it would look better if we remove underscore (_) before and after Learner, leader and Manager.